tx/textual/valuerenderer/message.go

```diff
@@ -0,0 +1,154 @@
+package valuerenderer
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type messageValueRenderer struct {
+	tr      *Textual
+	msgType protoreflect.ProtoMessage
+	fds     []protoreflect.FieldDescriptor
+}
+
+func NewMessageValueRenderer(t *Textual, msgType protoreflect.ProtoMessage) ValueRenderer {
```

Comment:
I would probably have done:

```suggestion
func NewMessageValueRenderer(t *Textual, msgDesc protoreflect.MessageDescriptor) ValueRenderer {
```

Reply:
And then the `protoreflect.FullName` -> `protoreflect.Message` resolving happens inside `Parse`, instead of inside valuerenderer.go

Reply:
Yeah, that's what I was thinking after dabbling with Any. Tried it, and it works quite well.